src/models/oryp8/README.md

```diff
@@ -0,0 +1,62 @@
+# Oryx Pro (oryp8)
+
+- [External Overview](./external-overview.md)
+- [Internal Overview](./internal-overview.md)
+- [Parts & Repairs](./repairs.md)
+
+![Oryx Pro](./img/oryp8.webp)
```

Comment:
Looks like this image still has the SD card reader. That should probably be updated